VaticanBAV/et/BAVet116.xml

```diff
@@ -67,132 +71,150 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                            <locus from="153r" to="160v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraJohnChrysostom"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.17">
-                            <locus from="160v" to="164r"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraDioscorus"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.18">
-                            <locus from="164r" to="171v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraJacobSerug"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.19">
-                            <locus from="171v" to="181r"/>
-                            <title type="complete" ref="LIT1098Anapho"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.20">
-                            <locus from="182v" to="186v"/>
-                            <title type="complete" ref="LIT2038Naakkw"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.21">
-                            <locus from="186v" to="194r"/>
-                            <title type="complete" ref="LIT1569habeka"/>
+
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2">
+                          <locus from="20v" to="24r"/>
+
+                          <msItem xml:id="ms_i1.2.1">
+                            <locus from="20v" to="22v"/>
+                            <title>Prayers of supplication for the Deceased</title>
                           </msItem>
-                          <msItem xml:id="ms_i1.22">
-                            <locus from="194r" to="197v"/>
-                            <title type="complete" ref="LIT2038Naakkw"/>
+
+                          <msItem xml:id="ms_i1.2.2">
+                            <locus from="22v" to="24r"/>
+                            <title>መስተብቍዕ፡</title>
                             <note>
-                              Repeated; added at the end is the prayer of the laying on of hands.
+                              Prayers before the reading of the Miracles of Our Lady Mary
                             </note>
                           </msItem>
+
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.3">
+                          <locus from="24r" to="29r"/>
+                          <title type="complete" ref="LIT2605Zayena"/>
+                          <note>With the litany written underneath</note>
                         </msItem>
+
+                        <msItem xml:id="ms_i1.4">
+                          <locus from="29v" to="69v"/>
+                          <title type="complete" ref="LIT1960Mashaf#SerataQeddase"/>
+                          <msItem xml:id="ms_i1.1.7.1">
+                            <locus from="29v" to="69v"/>
```

Comment:
check folio numbering

Reply:
what I meant is - is it correct that 1.4 and 1.4.1 cover the very same pages?

Reply:
This is my clarification. For ff. 29v-69v, the catalogue notes that the contents are the prayers for blessing the liturgical instruments, as in 108 ff. 16r-17r, followed by the general order of the missal. There's no specification as to where one begins and the other ends. I suppose the locus shouldn't be repeated before the sub-item, but I'm not sure which locus to give it, or whether to simply remove it and keep the general title for the Ser'ata Qeddase.

Reply:
if you know that the prayer is at the beginning of the text unit so you know the beginning but don't know the end then only fill out the `@from` but not the `@to`